source/_integrations/comelit.markdown

```diff
@@ -37,6 +37,40 @@ There is support for the following platform types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+{% configuration %}
```

Comment:
```suggestion
{% configuration_basic %}
```
For the integration with UI configuration, we usually use the configuration_basic block.